README.md

```diff
@@ -1 +1,154 @@
-# learn-web-development
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>README for a software traning website</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Software training website] <a name="about-project"></a>
+
+**[Portfolio project]** is a repository to an exercise in Microverse. This project helps student build a personal portfolio
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML
+> CSS
+> Javascript
+
+<details>
+  <summary>CLIENT</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+  
+    <ul>
+    <li><a href="https://www.w3schools.com/css/](https://www.w3schools.com/js/default.asp">Javascript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> A dynamic website for training software developers.
+
+- **[A mobile version]**
+- **[A desktop version]**
+- **[Dynamic card that displays the instuctors]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Click on this link to checkout the site.
+
+- [Live Demo Link](https://prius2055.github.io/learn-web-development/)
+
```

Comment:
- [ ] Dear @prius2055, As of now your live version only displayed a title, as part of the main requirements for this project we need to make sure the live version is working fine. 
> Please consider renaming your home.html to index.html. [read more here about what index.html file is](https://www.namecheap.com/support/knowledgebase/article.aspx/183/27/what-is-an-index-page/#:~:text=The%20index%20page%20is%20the,company%2C%20organization%2C%20or%20individual.).

![image](https://user-images.githubusercontent.com/69077061/225982904-c823a29b-9d8c-48d7-9220-2078b63ba849.png)